deeppavlov/core/commands/train.py

```diff
@@ -204,7 +217,7 @@ def _test_model(model: Component, metrics_functions: List[Tuple[str, Callable]],
 
     report = {
         'eval_examples_count': len(val_y_true),
-        'metrics': OrderedDict(metrics),
+        'metrics': OrderedDict(prettify_metrics(metrics)),
```

Comment:
prettify_metrics already returns `OrderedDict`

Reply:
I will fix it.